logistics/preparation.md

```diff
@@ -0,0 +1,109 @@
+# Preparation
+
+1. Identify the Preparation Owner from the Partner/Customer:
+    - Determine who the responsible party is on the customer/partner side.
+    - Ensure they possess:
+      - Proper Azure Access.
+      - Adequate knowledge about Azure.
+      - Authorization to configure subscriptions.
+    
+1. Define/Create the Azure Subscriptions for the Hackathon:
+**Warning**: Costs will incur. [Monitor/manage expenses](https://learn.microsoft.com/en-us/azure/cost-management-billing/cost-management-billing-overview).
+    - Given the increased access for participants, ensure you use [sandbox subscriptions](https://learn.microsoft.com/en-us/azure/architecture/guide/azure-sandbox/azure-sandbox).  
+    **Note**: No new tenant is required. Create a new subscription within the existing tenant.  
+    - Identify hackathon participants. Determine:
+        - Existing subscriptions
+        - Visual Studio subscriptions
+        - Existing dev/test (sandbox) subscriptions
+    - Collaborate with the preparation owner to determine the number of subscriptions required. Considerations:
+        - Number of teams or resources 
+        - **Recommendation**: Use a single sandbox subscription for all participants. Benefits of this approach include:
+            - Consistent Monitoring: Easier to track usage and costs across the board.
+            - Simplified Management: Reduces the administrative overhead of managing multiple subscriptions.
+            - Shared Resources: Facilitates seamless sharing of resources and information among teams.
+            - Cost Efficiency: Potentially lower costs as there are volume-based price benefits.
+    - Based on the chosen approach, define AD groups per team and create separate resource groups for each.
+    - Ensure participants have contributor access within their [specific resource groups](https://learn.microsoft.com/en-us/azure/role-based-access-control/quickstart-assign-role-user-portal).
+1. Request Access to Azure OpenAI Service for All Subscriptions    
+    - Visit [https://aka.ms/oai/access](https://aka.ms/oai/access). Submit details for each subscription.
+    - Monitor request status.   
+    **Note:** If your team requires other [limited access services](https://learn.microsoft.com/en-us/azure/ai-services/cognitive-services-limited-access) for the hack, you will need to request access for them.
+1. Create Shared Azure OpenAI Service Resources
+    - Initialize Azure OpenAI resources within each subscription
+    - Provision necessary resources (e.g., computation, storage)
+    - Document resource specifics for team access
+
+1. Deploy Models with Tokens Per Minute (TPM) Limits
```

Comment:
Same question as before: Should teams be able to deploy models themselves? The argument against this is that one team could use all the TPM for a region/subscription and block other teams.